zingolib/src/wallet/transactions.rs

```diff
@@ -789,7 +797,7 @@ impl TransactionMetadataSet {
             D::WalletNote::from_parts(
                 D::Recipient::diversifier(&to),
                 note.clone(),
-                Position::from(u64::MAX),
+                None,
                 Some(nullifier.unwrap_or_else(|| {
```

Comment:
Is this behavior creepy?   Just plug in 32 `1`'s...   seems risky.

Reply:
Uh....yeah let me fix this one

Reply:
fixed